group_vars/researchdata/production.yml

```diff
@@ -1,16 +1,28 @@
-db_host: "{{ vault_postgres_host }}"
+---
+db_host: "lib-postgres-prod3.princeton.edu"
 db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
-db_clusteradmin_user: "{{ vault_postgres_admin_user }}"
+db_clusteradmin_user: "postgres"
+
+desired_nodejs_version: "v8.17.0"
 
 postgres_host: "{{ db_host }}"
 postgres_port: "5432"
+pg_hba_postgresql_user: "all"
+pg_hba_postgresql_database: "all"
+pg_hba_method: "md5"
+pg_hba_source: "{{ ansible_host }}/32"
 postgres_admin_user: "{{ db_clusteradmin_user }}"
 postgres_admin_password: "{{ db_clusteradmin_password }}"
 postgresql_is_local: false
-postgres_version: 10
+postgres_version: 13
 drupal_db_host: '{{ postgres_host }}'
 drupal_db_name: 'researchdata_prod'
+drupal_db_user: "researchdata"
```

Comment:
@kayiwa why was this change needed?  It is defaulted in the role: https://github.com/pulibrary/princeton_ansible/blob/main/roles/researchdata/defaults/main.yml#L3

Reply:
I'm confused. the change on `group_vars/researchdata/production.yml` is there to ensure that we can do a dump and import of the file from `lib-postgres3` to `lib-postgres-prod3` the `playbooks/postgresql_db_migration.yml` file looks in there to determine what to dump. So it will not know of the existence of what is set up in `roles/researchdata/defaults/main.yml`

Reply:
Ok, this makes me sad that we can not have defaults in roles... But you are utilizing the group vars for something other than the role/playbook they were created for.